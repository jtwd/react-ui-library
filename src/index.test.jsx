import React from 'react'
import {shallow} from 'enzyme'
import Index from './index.jsx'

test('Index renders without crashing', () => {
  expect(JSON.stringify(Index)).toMatchSnapshot()
})
