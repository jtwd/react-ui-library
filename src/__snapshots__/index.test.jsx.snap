// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders without crashing 1`] = `"{}"`;
